src/v3/src/types/schema.ts

```diff
@@ -150,6 +153,8 @@ export type TokenReplacementValue = {
   attributes?: {
     class?: string;
     href?: string;
+    target?: string;
```

Comment:
```suggestion
    target?: '_self' | '_blank' | '_parent' | 'top';
```

You can even define a type and then replace anywhere in this file that uses `target`, i.e. 

type definition: 
`export type AnchorTargetType = '_self' | '_blank' | '_parent' | 'top';`

usage: 
`target?: AnchorTargetType`